radix-transaction-scenarios/assets/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+
+cd "$(dirname "$0")"
+
+IMAGE_NAME="radixdlt/scrypto-builder"
+IMAGE_TAG="v1.1.1"
+WORKSPACE_DIR="blueprints"
+DESTINATION_DIR="."
+for PACKAGE_NAME in "flash_loan" "global_n_owned" "kv_store" "max_transaction" "metadata" "radiswap" "royalties"
+do
+    # Run scrypto build 
+    docker run \
+        --platform=linux/amd64 \
+        --entrypoint=scrypto \
+        -v $(realpath $WORKSPACE_DIR):/src/$WORKSPACE_DIR \
+        $IMAGE_NAME:$IMAGE_TAG \
+        build --path /src/$WORKSPACE_DIR/$PACKAGE_NAME
+
+    # Copy artifacts
+    cp \
+        $WORKSPACE_DIR/target/wasm32-unknown-unknown/release/$PACKAGE_NAME.{wasm,rpd} \
+        $DESTINATION_DIR/
+done
+
+sha256sum *.{wasm,rpd}
+
+# The script should generate the following hashes:
```

Comment:
It's a little unclear to me what these hashes represent -- maybe we don't need them?